src/lib/yacc.rs

```diff
@@ -57,28 +59,41 @@ lazy_static! {
     static ref RE_TERMINAL: Regex = {
         Regex::new("^(?:(\".+?\")|('.+?')|([a-zA-Z_][a-zA-Z_0-9]*))").unwrap()
     };
-    static ref RE_WHITESPACE: Regex = {
-        Regex::new(r"^\s+").unwrap()
-    };
 }
 
 /// The actual parser is intended to be entirely opaque from outside users.
 impl YaccParser {
     fn new(src: String) -> YaccParser {
         YaccParser {
-            src    : src,
-            grammar: GrammarAST::new()
+            src     : src,
+            newlines: vec![],
```

Comment:
Really minor, but I personally prefer `Vec::new()` over `vec![]` for empty vectors.